app/src/main/java/org/simple/clinic/editpatient/OngoingEditPatientEntry.kt

```diff
@@ -112,47 +117,65 @@ data class OngoingEditPatientEntry @Deprecated("Use the `from` factory function
       numberValidator: PhoneNumberValidator,
       dobValidator: UserInputDateValidator
   ): Set<EditPatientValidationError> {
-    val errors = mutableSetOf<EditPatientValidationError>()
+    return getValidationChecks(alreadySavedNumber, numberValidator, ageOrDateOfBirth, dobValidator)
+        .mapNotNull { check -> check() }
```

Comment:
It was a little hard to understand what was happening here. Can we try changing this to `.mapNotNull { it.invoke() }` to see if it might be more understandable?